config/release-package-matrix.js

```diff
@@ -30,100 +50,375 @@ const krbConnTestsOpenSSL3 = ['node20', 'rocky9', 'ubuntu2204'];
+      },
+    ],
   },
   {
     executableOsId: 'linux-x64',
-    compileBuildVariant: 'linux_x64_build',
-    kerberosConnectivityTestDockerfiles: [...krbConnTestsOpenSSL11, ...krbConnTestsOpenSSL3],
+    compileBuildVariant: 'build_linux_x64',
+    kerberosConnectivityTestDockerfiles: [
+      ...krbConnTestsOpenSSL11,
+      ...krbConnTestsOpenSSL3,
+    ],
     packages: [
-      { name: 'linux-x64', description: publicDescriptions.linux_x64, packageType: 'tgz', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu20.04-tgz'], serverLikeTargetList: [...allLinux] },
-      { name: 'deb-x64', description: publicDescriptions.debian_x64, packageType: 'deb', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['ubuntu18.04-deb', 'ubuntu20.04-deb', 'ubuntu22.04-deb', 'ubuntu22.04-nohome-deb', 'ubuntu22.04-qemu-deb', 'ubuntu24.04-deb', 'debian10-deb', 'debian11-deb', 'debian12-deb'], serverLikeTargetList: [...ubuntu1804AndAboveAndDebBased] },
-      { name: 'rpm-x64', description: publicDescriptions.rhel_x64, packageType: 'rpm', packageOn: 'linux_package', smokeTestKind: 'docker', smokeTestDockerfiles: ['centos7-rpm', 'amazonlinux2-rpm', 'amazonlinux2023-rpm', 'rocky8-rpm', 'rocky9-rpm', 'fedora34-rpm', 'suse12-rpm', 'suse15-rpm', 'oraclelinux9-rpm'], serverLikeTargetList: [...rhel70AndAboveAndRpmBased] }
-    ]
+      {
+        name: 'linux-x64',
+        description: publicDescriptions.linux_x64,
+        packageType: 'tgz',
+        packageOn: 'linux_package',
+        smokeTestKind: 'docker',
+        smokeTestDockerfiles: ['ubuntu20.04-tgz'],
+        serverLikeTargetList: [...allLinux],
+      },
+      {
+        name: 'deb-x64',
+        description: publicDescriptions.debian_x64,
+        packageType: 'deb',
+        packageOn: 'linux_package',
+        smokeTestKind: 'docker',
+        smokeTestDockerfiles: [
+          'ubuntu18.04-deb',
+          'ubuntu20.04-deb',
+          'ubuntu22.04-deb',
+          'ubuntu22.04-nohome-deb',
+          'ubuntu22.04-qemu-deb',
+          'ubuntu24.04-deb',
+          'debian10-deb',
+          'debian11-deb',
+          'debian12-deb',
+        ],
+        serverLikeTargetList: [...ubuntu1804AndAboveAndDebBased],
+      },
+      {
+        name: 'rpm-x64',
+        description: publicDescriptions.rhel_x64,
+        packageType: 'rpm',
+        packageOn: 'linux_package',
+        smokeTestKind: 'docker',
+        smokeTestDockerfiles: [
+          'centos7-rpm',
+          'amazonlinux2-rpm',
+          'amazonlinux2023-rpm',
+          'rocky8-rpm',
+          'rocky9-rpm',
+          'fedora34-rpm',
+          'suse12-rpm',
+          'suse15-rpm',
+          'oraclelinux9-rpm',
+        ],
+        serverLikeTargetList: [...rhel70AndAboveAndRpmBased],
+      },
+    ],
   },
   {
     executableOsId: 'linux-x64-openssl11',
-    compileBuildVariant: 'linux_x64_build_openssl11',
+    compileBuildVariant: 'build_linux_x64',
```

Comment:
as far as I can see, this release matrix might need to be changed as some of these things are redundant right? from what I follow from the code, I am this change from i.e. `linux_x64_build_openssl11` to `build_linux_x64' shouldn't cause issues for the releases or anything and maybe this compileBuildVariant be moved from this matrix altogether.